sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFlux.java

```diff
@@ -58,6 +58,16 @@ public PagedFlux(Supplier<Mono<PagedResponse<T>>> firstPageRetriever,
         this.nextPageRetriever = nextPageRetriever;
     }
 
+    /**
+     * Creates an instance of {@link PagedFlux} using argument as a supplier that fetches the first page of {@code T}.
+
+     * <p><strong>Code sample</strong></p>
+     * @param firstPageRetriever Supplier that retrieves the first page
+     */
+    public PagedFlux(Supplier<Mono<PagedResponse<T>>> firstPageRetriever) {
```

Comment:
Does this actually help resolve the gap for non-paged fluxes? Or is this just a way to allow a consumer of the library to only request a single page?

Reply:
This constructor is removed.